mmseg/apis/ote/apis/segmentation/nncf_task.py

```diff
@@ -65,6 +66,7 @@ def __init__(self, task_environment: TaskEnvironment):
         self._val_dataloader = None
         self._compression_ctrl = None
         self._nncf_preset = "nncf_quantization"
+        self._optimization_type = ModelOptimizationType.NNCF
```

Comment:
Please, add this line after super().__init__(task_environment)

Reply:
Done, thanks!